bower.json

```diff
@@ -5,10 +5,16 @@
     "dist/rollbar.umd.js"
   ],
   "ignore": [
+    "dist/*.snippet*",
+    "dist/*.nojson*",
+    "dist/*.named-amd*",
```

Comment:
Sorry if I'm slow, but we're currently using the build from the Bower distribution. For example `path/to/bower/components/rollbar/dist/rollbar.named-amd.nojson.js`